command/quota_init.go

```diff
@@ -123,6 +123,10 @@ limit {
     cpu        = 2500
     memory     = 1000
     memory_max = 1000
+    device {
+      name  = "gpu/nvidia/1080ti",
```

Comment:
these are out of order - should be `nvidia/gpu/1080ti`

https://developer.hashicorp.com/nomad/docs/job-specification/device#-2

Reply:
yikes, good catch.

Reply:
I got really far making the same mistake :upside_down_face: